src/commands/convert/trivy2hdf.ts

```diff
@@ -21,10 +21,12 @@ export default class Trivy2HDF extends Command {
   async run() {
     const {flags} = await this.parse(Trivy2HDF)
     // comes as an _asff.json file which is basically the array of findings but without the surrounding object; however, could also be properly formed asff since it depends on the template used
-    let input = fs.readFileSync(flags.input, 'utf8').trim()
-    if (Array.isArray(JSON.parse(input))) {
-      input = `{"Findings": ${fs.readFileSync(flags.input, 'utf8').trim()}}`
-    }
+    const input = fs.readFileSync(flags.input, 'utf8').trim()
+    // if (Array.isArray(JSON.parse(input))) {
```

Comment:
did you test to confirm it works with both `{"Findings: [blah]}` as well as `[blah]` input types

Reply:
Yup! Seems like the ASFF mapper is able to deal with that now.